translate/src/context/TranslationContext.tsx

```diff
@@ -59,7 +59,10 @@ export const LLMTranslationProvider: React.FC = ({ children }) => {
       stateRef.current.set(mt, {
         loading: false,
         selectedOption: characteristic,
```

Comment:
It's a bit confusing that "selected option" and "characteristic" are names for essentially the same value, which is typed only as a `string`. These values should be typed more strictly, with something like
```ts
type Characteristic = 'rephrased' | 'formal' | 'informal'
```

Also, the variable names should be consistent, so `selectedCharacteristic` instead of `selectedOption`.